styles/_base.scss

```diff
@@ -119,7 +119,6 @@ hr {
     left: 0;
     right: 0;
     bottom: 0;
-    background: rgba(0, 0, 0, 0.3);
```

Comment:
I like that while your entering time the rest of the page is unclickable, personally. But I think we should implement pressing enter and escape closes out the timer.